### ISSUE TITLE
Adding skip-option for singledispatch and overloaded functions.

### ISSUE THREAD

**Issue by vacarme** [HunterMcGushion/docstr_coverage#125]
Hello.
May what do you guys think of adding some skip options which seems to me relevant and not driven by docstring-fatigue.

+ `@functools.singledispatch`
+ `@overloaded`

Both tends to make code longer to read, sometimes docstring usage could be relevant for each signatures but in some cases a docstring below the first signature is enough, because the behavior Is self-explanatory.

`@functools.singledispatch` could be to ignore function named `_`.

**Comment by MiWeiss** [comment]
Hi @vacarme 

Sounds reasonable, but I'm afraid that adding too many special ignore flags will eventually make docstr-coverage too verbose.

Maybe we could thinkg about some multi-value argument, e.g, `--ignore-annotation @some @annotation @and @more` but I'd have to think about how to eventually do this (e.g. how to treat different imports, e.g. `@functools.singledispatch` vs. `from functools import singledispatch` followed by `@singledispatch`.

@HunterMcGushion What do you think?

**Comment by HunterMcGushion** [comment]
I agree with @MiWeiss. This does sound like a useful option, but I'm concerned about adding too many flags. @vacarme, would simply  `--exclude`ing `_` be sufficient in this case?

**Comment by MiWeiss** [comment]
I'm opening this up for anyone to come up with a smart proposal on how something along the lines of `--ignore-annotation @some @annotation @and @more` could be achieved.
